clients/src/test/java/org/apache/kafka/common/security/authenticator/SaslAuthenticatorTest.java

```diff
@@ -471,7 +471,7 @@ public void testUnauthenticatedApiVersionsRequestOverSslHandshakeVersion0() thro
      */
     @Test
     public void testUnauthenticatedApiVersionsRequestOverSslHandshakeVersion1() throws Exception {
-        testUnauthenticatedApiVersionsRequest(SecurityProtocol.SASL_PLAINTEXT, (short) 1);
+        testUnauthenticatedApiVersionsRequest(SecurityProtocol.SASL_SSL, (short) 1);
```

Comment:
Why do we need this change?

Reply:
That was a typo in the code, SASL_PLAINTEXT is tested a few lines above this.